src/shared/sam/activation.ts

```diff
@@ -98,6 +100,7 @@ async function registerServerlessCommands(params: {
             await createNewSamApplication(params.channelLogger, params.awsContext, params.regionProvider)
         }),
         vscode.commands.registerCommand('aws.configureLambda', configureLocalLambda),
+        vscode.commands.registerCommand('aws.addSamDebugConfiguration', addSamDebugConfiguration),
```

Comment:
Looks like these registerComand calls don't have their disposable-based retVals handled for clean-up. Could you please add that?

Reply:
they are part of `subscriptions.push()` call on line 93 which is variadic.

Reply:
Thanks @justinmk3 , I keep getting fooled by that 😆